content/en/building/workflows/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Workflows
+linkTitle: Workflows
+weight: 3
+description: >
+  Overview and configuration of various workflows in CHT Applications.
```

Comment:
```suggestion
  Overview and configuration of workflows in CHT Applications
```

Reply:
We are not going to have Concepts section within the docs-site once migration is complete